.gitignore

```diff
@@ -2,3 +2,30 @@ node_modules/
 !dist/
 .repository/
 .idea
+acorn
```

Comment:
What's with all these executables that needs to be ignored? If everything is setup correctly they live in `node_modules/.bin` and will be excluded already.

Reply:
On Windows, they got added to the root folder. Since we do not have any guidelines for the development of actions on Windows, I would assume this is safer and less hassle for new development environments.

Reply:
There must be something else that's causing this problem.